thirdparty/KDDockWidgets/src/private/WidgetResizeHandler.cpp

```diff
@@ -128,6 +129,14 @@ bool WidgetResizeHandler::eventFilter(QObject *o, QEvent *e)
         updateCursor(CursorPosition_Undefined);
         auto mouseEvent = static_cast<QMouseEvent *>(e);
 
+        // When we stop dragging a floating window which has a single dock widget, we save the position
```

Comment:
It also turned out that resizing the window wouldn't update the saved rect, so I added this.